fetch/client/src/main/scala/endpoints4s/fetch/Endpoints.scala

```diff
@@ -31,7 +31,6 @@ trait EndpointsWithCustomErrors
     with StatusCodes {
 
   def settings: EndpointsSettings
-  implicit def ec: ExecutionContext
```

Comment:
We actually did not notice this when finalizing https://github.com/endpoints4s/endpoints4s/pull/934 - after the removal of conversions of `Promise` to `Future`, the `ExecutionContext` became unused.